### DISCUSSION TITLE
2024/10/15/hello-hexos/

### DISCUSSION THREAD

**Discussion by giscus** [Apricityx/apricityx.github.io#2] [Announcements]
# 2024/10/15/hello-hexos/

搭建个人博客 教程来源： https://xie.infoq.cn/article/ac51ce1f6e9434779c35cbb6c

https://apricityx.github.io/2024/10/15/hello-hexos/

**Comment by Apricityx** [comment]
**Comment test**

**Comment by Apricityx** [comment]
你好
